src/tool/subcommands/api_cmd.rs

```diff
@@ -340,7 +346,7 @@ impl RpcTest {
             }),
             check_semantics: Arc::new(|_, _| true),
             ignore: None,
-            pass_on_rejected: false,
+            policy_on_rejected: PolicyOnRejected::Fail,
```

Comment:
Should we make `PolicyOnRejected::PassWithIdenticalError` the default policy?

Reply:
> Should we make `PolicyOnRejected::PassWithIdenticalError` the default policy?

Good question. 

If we define the endpoint wrong, sometimes both lotus and forest fail for all the test iterations. I don't think this should result in a green checkmark. 

An alternative approach here would be to make sure that we have at least 1 passing iteration, otherwise consider it a failure altogether.